src/callbacks.py

```diff
@@ -193,17 +196,19 @@ def update_geofigure(selected_year, selected_type, selected_value):
                 alt.value("grey"),
                 alt.Color(
                     f"{y_column}:Q",
-                    title=f"{'Incidence' if selected_value == 'incidence' else 'Mortality'} {'Absolute' if selected_type == 'absolute' else 'Relative'}",
-                    legend=alt.Legend(
-                        orient="none",
-                        # direction="horizontal",
-                        titleAnchor="middle",
-                    ),
+                    title=f"{'Absolute' if selected_type == 'absolute' else 'Relative'} {'Incidence' if selected_value == 'incidence' else 'Mortality'}",
                     scale=alt.Scale(scheme="plasma"),
+                    legend=alt.Legend(orient="none", titleAnchor="middle"),
                 ),
             ),
-            tooltip=["country:N", f"{y_column}:Q"],
-            opacity=opacity,
+            tooltip=[
+                alt.Tooltip("country:N", title="Country"),
+                alt.Tooltip(
+                    f"{y_column}:Q",
+                    title=f"{'Absolute' if selected_type == 'absolute' else 'Relative'} {'Incidence' if selected_value == 'incidence' else 'Mortality'}",
+                ),  # Corrected dynamic variable reference and format
+            ],
+            opacity=alt.condition(highlight, alt.value(0.8), alt.value(0.5)),
```

Comment:
This can just be the following, since we defined this in a variable
```suggestion
            opacity=opacity
```

Reply:
Thanks Sean! Fixed it